public_html/assets/css/nf-core.css

```diff
@@ -781,7 +784,7 @@ Based on https://codepen.io/wintr/pen/beBJBb */
   margin-top: .25rem;
 }
 
-.side-sub-subnav {
+.pipeline-page-content, .side-sub-subnav {
     padding-top: 50px;
```

Comment:
I don't like the empty space it creates when scrolled to the top. 
<img width="1160" alt="Screenshot 2020-07-24 at 14 12 59" src="https://user-images.githubusercontent.com/6169021/88390133-151b2b00-cdb8-11ea-87e2-2127f65b1efe.png">

Can we switch this to a scrollpoint based behavior?

Reply:
What do you mean sorry?

Reply:
Can we add the padding above `.pipeline-page-content, .side-sub-subnav` only when it is actually needed, that is when the navbar would overlap them.

Reply:
aha, yes - sorry was probably not looking at this page when I added that. I think that the Usage docs needed padding (can go elsewhere though).